pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -413,6 +430,10 @@ func Run() {
 			// XXX add file watch...
 			checkAndPublishDhcpLeases(&zedrouterCtx)
 
+		case <-flowStatTimer.C:
+			log.Infof("**FlowStatTimer at %v", time.Now())
```

Comment:
Better to downgrade the periodic messages to log.Debugf

Reply:
ok.